api/src/paths/project/{projectId}/survey/create.ts

```diff
@@ -260,3 +272,23 @@ export const insertSurveyPermit = async (
 
   return true;
 };
+
+export const insertSurveyFundingSource = async (
+  funding_source_id: number,
+  survey_id: number,
+  connection: IDBConnection
+): Promise<boolean> => {
+  const sqlStatement = insertSurveyFundingSourceSQL(survey_id, funding_source_id);
+
+  if (!sqlStatement) {
+    throw new HTTP400('Failed to build SQL statement for insertSurveyFundingSource');
+  }
+
+  const response = await connection.query(sqlStatement.text, sqlStatement.values);
+
+  if (!response) {
+    throw new HTTP400('Failed to insert survey funding source data');
+  }
+
+  return true;
```

Comment:
I know we have this in a few spots to help with testing, but we shouldn't need to return something here if the actual function doesn't need to.

Rather than assert it returns true in the tests, we can just be satisfied if no errors are thrown.  
Or if you want to go one step further, assert that the mock connection query object was called 1 time (and optionally assert what values it was called with).

Reply:
ok sounds good